test/experimental/shortestpaths.jl

```diff
@@ -494,7 +494,7 @@ end
             add_edge!(G, 1, 3)
             m = float([0 2 2 0 0 1; 2 0 1 0 0 0; 2 1 0 4 0 0; 0 0 4 0 1 0; 0 0 0 1 0 1; 1 0 0 0 1 0])
 
-            for g in testgraphs(G)
+            @testset "$g" for g in testgraphs(G)
```

Comment:
testset should specify what SP algorithm we're testing.

Reply:
Doesn't the parent testset do this sufficiently here?